test_scripts/Defects/6_1/885_For_some_RPCs_Core_does_not_unsubscribe_from_softbuttons_after_receiving_a_response/AlertManeuver.lua

```diff
@@ -0,0 +1,111 @@
+-- 1) For all of the popup-based RPCs which contain softButtons, Core is supposed to stop processing softButton events once a response is received for the RPC.
+-- Currently the only one that does this is ScrollableMessage, for all of the others Core stays subscribed to their softButtons after receiving a response.
+-- Actual result:
+-- 1)
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('user_modules/sequences/actions')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Functions ]]
+local function pTUpdateFunc(tbl)
+    local VDgroup = {
+        rpcs = {
+            AlertManeuver = {
+                hmi_levels = { "BACKGROUND", "FULL", "LIMITED" }
+            }
+        }
+    }
+    tbl.policy_table.functional_groupings["NewTestCaseGroup"] = VDgroup
+    tbl.policy_table.app_policies[config.application1.registerAppInterfaceParams.fullAppID].groups = {"Base-4",
+    "NewTestCaseGroup"}
+end
+
+local function AlertManeuver()
+    local params = {
+        ttsChunks = {
+          { type = "TEXT", text = "pathToFile" }
+        },
+        softButtons = {
+            { type = "TEXT", softButtonID = 4, text = "text" }
+        }
+    }
+    local responseDelay = 3000
+    local cid = common.getMobileSession():SendRPC("AlertManeuver", params )
+    EXPECT_HMICALL("Navigation.AlertManeuver",{
+        softButtons = {
+            {type = "TEXT",
+            softButtonID = 4}
+        }
+    })
+    :Do(function(_, data)
+        local function alertManeuverResponse()
+            common.getHMIConnection():SendResponse(data.id, "Navigation.AlertManeuver", "SUCCESS", { })
+        end
+        RUN_AFTER(alertManeuverResponse, responseDelay)
+    end)
+    params.appID = common.getHMIAppId()
+    common.getHMIConnection():ExpectRequest("TTS.Speak", {
+        ttsChunks = {
+          { type = "TEXT", text = "pathToFile" }
+        }
+    })
+	:Do(function(_,data)
+		common.getHMIConnection():SendNotification("TTS.Started")
+		local function SpeakResponse()
+			common.getHMIConnection():SendResponse(data.id, "TTS.Speak", "SUCCESS", { })
+			common.getHMIConnection():SendNotification("TTS.Stopped")
+		end
+		local function ButtonEventPress()
+            common.getHMIConnection():SendNotification("Buttons.OnButtonEvent",
+			{ name = "CUSTOM_BUTTON", mode = "BUTTONDOWN", customButtonID = 4, appID = common.getHMIAppId()
+			})
+            common.getHMIConnection():SendNotification("Buttons.OnButtonEvent",
+			{ name = "CUSTOM_BUTTON", mode = "BUTTONUP", customButtonID = 4, appID = common.getHMIAppId()
+			})
+            common.getHMIConnection():SendNotification("Buttons.OnButtonPress",
+			{ name = "CUSTOM_BUTTON", mode = "SHORT", customButtonID = 4, appID = common.getHMIAppId()
+			})
+		end
+
+		RUN_AFTER(ButtonEventPress, 1000)
```

Comment:
I think to test the fix the ButtonPress events should happen both before and after the Alert response is received https://github.com/smartdevicelink/sdl_core/pull/2629/files#diff-6df67a58b10c424cbd1522799ebe45c0a4aa71377338edb779e81bbf8601601cR64 to check that the unsubscription was successful.